index.js

```diff
@@ -419,6 +419,21 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a unary function and transforms it into a unary function in another
+  //. setting.
+  //.
+  //. ```javascript
+  //. > S.lift(R.add(1))(S.Just(2))
+  //. S.Just(3)
+  //. > S.lift(R.add(1))(S.Nothing())
+  //. S.Nothing()
+  //. ```
```

Comment:
Please include an empty—well, `//.`—line between doctests.

Let's use `R.inc` rather than `R.add(1)`.

Let's use the `S.lift(f, x)` form rather than the `S.lift(f)(x)` one. This highlights a difference between `R.lift` and `S.lift`.